src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,41 @@
-export const Tabs = () => {};
+/*eslint-disable*/
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  if (selectedTabId === 'asdasd') {
+    selectedTabId = tabs[0].id;
+  }
+
+  const activeTab = tabs.find(tab => tab.id === selectedTabId);
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              data-cy="Tab"
+              className={selectedTabId === tab.id ? 'is-active' : ''}
+            >
+              <a
+                onClick={() => {
+                  if (selectedTabId !== tab.id) {
+                    onTabSelected(tab);
+                  }
+                }}
```

Comment:
```suggestion
                onClick={() => selectedTabId !== tab.id && onTabSelected(tab)}
```

can be more simplify